src/main/java/org/jabref/gui/preferences/customentrytypes/CustomEntryTypesTab.java

```diff
@@ -176,7 +177,7 @@ private void setupFieldsTable() {
         makeRotatedColumnHeader(fieldTypeColumn, Localization.lang("Required"));
 
         fieldTypeMultilineColumn.setCellFactory(CheckBoxTableCell.forTableColumn(fieldTypeMultilineColumn));
-        fieldTypeMultilineColumn.setCellValueFactory(item -> item.getValue().multilineProperty());
+        fieldTypeMultilineColumn.setCellValueFactory(this::createMultilinePropertyListener);
```

Comment:
There should be a comment here - or a JavaDoc comment on the method, why this is necessary.

I see that ` fieldTypeColumn.setCellValueFactory` is the same code as the old code here - and I don't understand why `required` works, but `multiline` not.

Reply:
Hey @koppor, added a comment there as you suggested on its necessity on commit https://github.com/JabRef/jabref/pull/12040/commits/830b2e4565fc6fcd6f1cb28a8afa3c8bee5cd31f. In the original code, required was working and multiline was not working as required is per entry but multiline is global. Due to this, if all the checkboxes were not unchecked in all entry types then it it would not uncheck it. Rather than replacing the original logic in the [storeSettings](https://github.com/JabRef/jabref/blob/main/src/main/java/org/jabref/gui/preferences/customentrytypes/CustomEntryTypesTabViewModel.java#L102) method I opted to add a listener that updates all other entry types multiline checkbox fixing the original issue and also giving the user visual feedback of the change.